site/pages/about.md

```diff
@@ -0,0 +1,41 @@
+<!--
+.. title: About the NCAR Experimental Development Team
+.. slug: about
+.. date: 2019-10-21 15:46:50 UTC-06:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+The NCAR Experimental Development (Xdev) Team is a small team of software engineers
+dedicated to the advancement and development of modern tools and techniques
+in scalable data analysis for geoscientists.  The Xdev Team was created as part
+of a collaboration with the [Pangeo](https://pangeo.io) community.
+
+Currently, the Xdev Team consists of:
+
+- **Kevin Paul (Engineering Lead)**: Kevin is a scientist and software engineer at NCAR with a
+  specialization in developing parallel Python tools for data post-processing, most recently developing
+  tools to post-process the CESM contributions for CMIP6.
+- **Matthew Long (Science Lead)**: Matt is a Scientist in the Climate & Global Dynamics laboratory
+  at NCAR with research foci in Earth system modeling, ocean ecosystems, and biogeochemistry.
+- **Anderson Banihirwe**: Anderson is a software engineer at NCAR with a deep understanding
+  of the Python ecosystem, modern software best practices, and modern data analysis.
+- **Julia Kent**: Julia has a background in atmospheric and oceanic sciences and has recently
+  transitioned into software engineering with a specialty in working with scientists to develop
+  Python-based solutions to their workflows.
+- **Brian Bonnlander**: Brian is a computer scientist with a specialty in machine learning and
```

Comment:
"to provide high-level data analysis tools that allow scientists to focus on science, rather than software engineering."